cmd/antrea-agent/options.go

```diff
@@ -31,7 +31,7 @@ const (
 	defaultHostGateway        = "gw0"
 	defaultHostProcPathPrefix = "/host"
 	defaultServiceCIDR        = "10.96.0.0/12"
-	defaultMTUVxlan           = 1450
+	defaultMTUVXLAN           = 1450
```

Comment:
If the value of defaultMTUVXLAN and defaultMTUGeneve are the same, maybe we could use a single constant?

Reply:
The code is by @antoninbas. I guess he just wants to make it more explicit.

Reply:
I don't think there will ever be a difference between VxLAN And Geneve, but I don't think using 1 or 2 constants makes any sort of difference. It does not impact readability in a negative way for sure.

Reply:
IIRC the encap size for VXLAN and NVGRE is the same, so we could use one constant. So I'm fine with having a single `defaultMTUEncap` constant, although I think the 2 constants are just as good for readability's sake. If you go with one constant, please add a comment stating that header size is the same for the 2 tunnel protocols we support.

Reply:
Are there some cases that the ofportRequest should be 0/-1, or I should say the caller expects OVS to assign ofport number automatically? One case I can think about is we might have multiple tunnel ports after enable IPSec, and let OVS to descides the ofport number should be better than Antrea client assign it.